info.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/golang/gddo/gosrc"
+)
+
+type defInfo struct {
+	Package  string
+	Path     string
+	Repo     string
+	Unit     string
+	UnitType string
+	Kind     string
+}
+
+func outputData(data ...interface{}) {
+	output := fmt.Sprintln(data...)
+	if !*useJSON {
+		fmt.Print(output)
+		return
+	}
+	printStructured(output)
+}
+
+func printStructured(output string) {
+	// TODO main package files
+	datas := strings.Split(strings.Trim(output, "\n"), " ")
+	info := defInfo{
+		UnitType: "GoPackage",
+	}
+	if len(datas) > 0 {
+		info = setRepo(info, datas[0])
+	}
+	if len(datas) > 1 {
+		info.Path = datas[1]
+	}
+	if len(datas) > 2 {
+		field := datas[2]
+		info.Path = info.Path + "/" + field
+	}
+	if info.Path == "" {
+		info.Kind = "package"
+	}
+	json.NewEncoder(os.Stdout).Encode(info)
+}
+
+func setRepo(info defInfo, fullPath string) defInfo {
+	info.Unit = fullPath
+	if gosrc.IsGoRepoPath(fullPath) {
```

Comment:
this introduces an external repo dependency, which godefinfo was designed to avoid. just inline the IsGoRepoPath func (and add a notice attributing it to gddo for license purposes)